api/project/v1/project.proto

```diff
@@ -22,9 +22,9 @@ service ProjectAPI {
 message GetProjectsRequest {
   repeated string projects = 1;
   repeated string users = 2;
-  // If this is set to true, we will compute all project dependencies for the requested projects.
+  // If this is set to true, the api will return all project dependencies for the requested projects.
```

Comment:
and to clarify dependencies, it's immediate upstreams and immediate downstreams, right?

Reply:
Yeah I can link to the project proto here to make it more clear, but the literal dependencies field on the proto which does include all up/downstream.

https://github.com/lyft/clutch/blob/2304adf0e0189734d8a36803964214ee3bc73fbc/api/core/project/v1/project.proto#L19